bundleserver/src/main/java/com/ddd/server/repository/entity/ServerWindow.java

```diff
@@ -0,0 +1,68 @@
+package com.ddd.server.repository.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity(name = "ServerWindow")
+@Table(name = "serverwindow")
+public class ServerWindow {
+    @Id
+    @Column(name = "clientID", nullable = false, columnDefinition = "VARCHAR(256)")
+    private String clientID;
+
+    @Column(name = "startCounter", nullable = false, columnDefinition = "VARCHAR(256)")
```

Comment:
everything other than clientID is nullable, and is in fact default null

you can use SQL command, **show create table <tablename>** OR **desc <tablename>** to verify columns description